mono_repo/test/dart_command_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:mono_repo/src/commands/dart.dart';
+import 'package:mono_repo/src/commands/pub.dart';
+import 'package:mono_repo/src/root_config.dart';
+import 'package:path/path.dart' as p;
+import 'package:test/test.dart';
+import 'package:test_descriptor/test_descriptor.dart' as d;
+
+void main() {
+  group('valid setup', () {
+    setUp(_setup);
+
+    test('can run pub get', () async {
```

Comment:
Note that the `pub` command isn't directly tested any longer - it isn't really very feasible to do so any longer (except through a true integration test which we don't have infra for).

But, it also isn't really needed (all it does is pre-pend `pub`).